janusgraph-cql/src/main/java/org/janusgraph/diskstorage/cql/CQLConfigOptions.java

```diff
@@ -125,7 +125,8 @@
             "compact-storage",
             "Whether the storage backend should use compact storage on tables. This option is only available for Cassandra 2 and earlier and defaults to true.",
             ConfigOption.Type.FIXED,
-            Boolean.class);
+            Boolean.class,
+            true);
```

Comment:
Do we want this as default with new Janusgraph version?

Reply:
Unless we drop support for Cassandra 2, we should leave this defaulting to true. It's a sensible default for Cassandra 2 and doesn't get applied in Cassandra 3. 

Here's a [link to the logic](https://github.com/pluradj/janusgraph/blob/692df5dbc7587c99cc99c18db71729cee5286b64/janusgraph-cql/src/main/java/org/janusgraph/diskstorage/cql/CQLKeyColumnValueStore.java#L229) that makes sure it only initializes a table with compact storage if the Cassandra major version is less than 3 and the config option is true. 

This is really just a small fix for PR #1135.

Reply:
Thanks for the info